WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import { Dropdown, Header, Table } from 'semantic-ui-react';
+import I18n from '../../../lib/i18n';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import { apiV0Urls } from '../../../lib/requests/routes.js.erb';
+import Errored from '../../Requests/Errored';
+import Loading from '../../Requests/Loading';
+
+function GroupTable({ groupId }) {
+  const { data: roles, loading, error } = useLoadedData(apiV0Urls.userRoles.listOfGroup(
+    groupId,
+    { isActive: true, isGroupHidden: false, status: 'leader' },
+  ));
+
+  if (loading || loading) return <Loading />;
+  if (error) return <Errored />;
+
+  return (
+    <>
+      <Header as="h2">Active members</Header>
+      <Table celled>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>Name</Table.HeaderCell>
+            <Table.HeaderCell>Status</Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {roles.map((role) => (
```

Comment:
Might be good to sort the array, maybe first by status and then alphabetically?

Reply:
Sorting looks like a good idea to sort. I feel giving priority to start_date over name would be also good. I've done the sorting now.

I also felt that sorting in server side would be better based on the sort parameter. I've done that now.